setup.py

```diff
@@ -589,7 +594,8 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
     'jdbc': jdbc,
     'jira': jira,
     'kerberos': kerberos,
-    'kubernetes': kubernetes,  # TODO: remove this in Airflow 2.1
+    'kubernetes': kubernetes,   # TODO: remove this in Airflow 2.1
+    'kylin': kylin,
```

Comment:
It's duplicate. I think `airflow.kylin` is enough.

Reply:
Thank you very much! Should the key is airflow.kylin or apache.kylin?

Reply:
`apache.kylin`.

Reply:
Hi  mik-laj，The test  step of  ‘CI Build / Backport packages (pull_request) ’   still not pass，can you give me some suggestion? Thanks!